src/main/java/ru/skypro/homework/controller/UserController.java

```diff
@@ -1,55 +1,88 @@
 package ru.skypro.homework.controller;
 
 import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
 import lombok.AccessLevel;
 import lombok.RequiredArgsConstructor;
 import lombok.experimental.FieldDefaults;
-import org.springframework.http.HttpStatus;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 import org.springframework.web.multipart.MultipartFile;
 import ru.skypro.homework.dto.NewPasswordDTO;
 import ru.skypro.homework.dto.UpdateUserDTO;
-import ru.skypro.homework.dto.UserDto;
+import ru.skypro.homework.dto.UserDTO;
+import ru.skypro.homework.service.impl.UserServiceImpl;
+import ru.skypro.homework.service.entities.UserEntity;
+import ru.skypro.homework.service.repositories.UserRepository;
 
-import javax.transaction.Transactional;
+import java.util.Optional;
 
+@Slf4j
 @CrossOrigin(value = "http://localhost:3000")
 @RequiredArgsConstructor
 @FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
 @RestController
 @RequestMapping("/users")
 public class UserController {
 
+    UserServiceImpl userService;
```

Comment:
```suggestion
    UserService userService;
```
инъекцию надо делать по интерфейсам, а не реализациям